flint.ui/src/store/modules/rothc.js

```diff
@@ -65,173 +67,204 @@ export default {
+            evapoOpenRatio: "$#2.5$#",
+            bToCMaxTSMDRatio: "$#3.0$#"
           }
         }
       ],
       Modules: {
         TransactionManagerEndOfStepModule: {
-          library: 'internal.flint',
+          library: "internal.flint",
           order: 1
         },
         CalendarSequencer: {
-          library: 'internal.flint',
+          library: "internal.flint",
           order: 2
         },
         PlantResidueModule: {
           enabled: true,
-          library: 'moja.flint.example.rothc',
+          library: "moja.flint.example.rothc",
           order: 3
         },
         SoilCoverModule: {
           enabled: true,
-          library: 'moja.flint.example.rothc',
+          library: "moja.flint.example.rothc",
           order: 4
         },
         RothCModule: {
           enabled: true,
-          library: 'moja.flint.example.rothc',
+          library: "moja.flint.example.rothc",
           order: 5
         },
         ErrorScreenWriter: {
           enabled: true,
-          library: 'moja.flint.example.base',
+          library: "moja.flint.example.base",
           order: 10,
           settings: {
             log_errors_max_number: 10
           }
         },
         OutputerStream: {
           enabled: true,
-          library: 'internal.flint',
+          library: "internal.flint",
           order: 11,
           settings: {
-            output_filename: 'Example_Rothc_Point_Stock.csv',
+            output_filename: "Example_Rothc_Point_Stock.csv",
             output_to_screen: true,
             output_info_header: true
           }
         },
         OutputerStreamFlux: {
           enabled: true,
-          library: 'internal.flint',
+          library: "internal.flint",
           order: 12,
           settings: {
-            output_filename: 'Example_Rothc_Point_Flux.csv',
+            output_filename: "Example_Rothc_Point_Flux.csv",
             output_to_screen: false,
             output_info_header: true
           }
         }
       }
-    }
+    },
+    results: {},
+    DPM: [],
+    RPM: [],
+    BIO: [],
+    HUM: [],
+    IOM: []
   },
 
   mutations: {
+    update_DPM(state, DPM) {
+      console.log('updated DPM in state')
+      console.log(this.state.rothc.DPM)
+      Vue.set(state, 'DPM', DPM)
+    },
+    update_RPM(state, RPM) {
+      console.log('updated RPM in state')
+      console.log(this.state.rothc.RPM)
+      Vue.set(state, 'RPM', RPM)
+    },
+    update_BIO(state, BIO) {
+      console.log('updated BIO in state')
+      console.log(this.state.rothc.BIO)
+      Vue.set(state, 'BIO', BIO)
+    },
+    update_HUM(state, HUM) {
+      console.log('updated HUM in state')
+      console.log(this.state.rothc.HUM)
+      Vue.set(state, 'HUM', HUM)
+    },
+    update_IOM(state, IOM) {
+      console.log('updated IOM in state')
+      console.log(this.state.rothc.IOM)
+      Vue.set(state, 'IOM', IOM)
+    },
     setNewConfig_dpmaCMInit(state, newValue) {
```

Comment:
* `setNewConfig_*` -> `set_*` (for all related getters and setters just use the label`

Reply:
@waridrox is this fine?